File: Python/categorize_new_member.py
# mine
def open_or_senior(data):
    result = []
    for index, (age, handicap) in enumerate(data):
        if age > 54 and handicap > 7:
            result.append("Senior")
        else:
            result.append("Open")
    return result

# pythonic
def openOrSenior(data):
  return ["Senior" if age >= 55 and handicap >= 8 else "Open" for (age, handicap) in data]

File: JavaScript/categorize-new-member.js
# mine
function openOrSenior(data){
  let result = []
  for (const [age, handicap] of data) {
    if (age > 54 && handicap > 7) {
      result.push("Senior")
    } else {
      result.push("Open")
    }
  }
  return result
}

# better
function openOrSenior(data){
  return data.map(([age, handicap]) => (age > 54 && handicap > 7) ? 'Senior' : 'Open');
}
